CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_quickbooks v0.5.1
+## Bug Fixes 🐛🪛
+- Created indices for `double_entry_transactions` models. Used row_number functions for `payment`, `bill_payment` and `transfer` models.
```

Comment:
Should we call out that people will see duplicate indices since we are creating double entries for these records? This may be helpful since it could be confusing that there are duplicates.